Extensions/Tracing-Zipkin.md

```diff
@@ -0,0 +1,55 @@
+# Tracing (Zipkin) Metadata Extension
+
+_This extension specification is currently incubating._
+
+## Introduction
+Observability and tracing are key requirements for robust and reliable applications.  When using distributed applications connected with RSocket, it's important to propagate metadata about the current logical operations throughout the entire system.  One of the most popular systems for doing this kind of tracing is [Zipkin][z].  This extension specification provides an interoperatble structure for Zipkin metadata payloads to contain tracing information.  It is designed such that systems can efficently communicate span and trace information to a Zipkin server and propagate that information throughout a distributed system.
+
+[z]: https://zipkin.io
+
+## Metadata Payload
+This metadata type **MUST** only be used in frame types used to initiate streams and payloads.  This includes [`REQUEST_FNF`][rf], [`REQUEST_RESPONSE`][rr], [`REQUEST_STREAM`][rs], [`REQUEST_CHANNEL`][rc], and [`PAYLOAD`][p].  The Metadata MIME Type is `message/x.rsocket.tracing+zipkin`.
+
+[p]:  ../Protocol.md#frame-payload
+[rc]: ../Protocol.md#frame-request-channel
+[rf]: ../Protocol.md#frame-fnf
+[rr]: ../Protocol.md#frame-request-response
+[rs]: ../Protocol.md#frame-request-stream
+
+### Metadata Contents
+```
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |            Version            |P|S|D|     Flags   |
```

Comment:
I think after reading yuri's previous comments the version isn't needed because of the mime-type

Reply:
I'll update this once I get confirmation on [this](
https://github.com/rsocket/rsocket/pull/255#discussion_r227130485).